recipe/meta.yaml

```diff
@@ -1,130 +1,21 @@
-{% set version = "7.0.0" %}
-{% set python_min = python_min | default("3.11") %}
-{% set min_py = python_min | replace('.', '') | int %}
-{% set py = py | default(min_py) %}
+{% set version = old_astropy_version | default("6.1.6") %}
```

Comment:
what happened with 6.1.7?

Reply:
(I suppose it's unrelated to this PR, and it got caught up with missing an auto-triggered PR here?)

Reply:
Maybe, also this version just needs to be any valid version to make the linter happy  all the actual versions get pulled in from `conda_build_config.yaml`